src/applications/disability-benefits/view-payments/components/responsive-table/ResponsiveTable.scss

```diff
@@ -0,0 +1,58 @@
+table.responsive {
+  font-family: 'Open Sans', sans-serif;
+  border-collapse: collapse;
+  margin: 0;
+  padding: 0;
+  width: 100%;
+}
+
+table.responsive tr {
+  padding: 0.35em;
+}
+
+table.responsive td {
+  word-wrap: break-word;
+}
+
+@media screen and (max-width: 600px) {
```

Comment:
Wondering if there needs to be some additional tweaking to the mobile version of this table. Right at 599px, this feels like too much negative space. I think the padding for the `Payments you received` header and subtext may need to be adjusted as well. I couldn't find mobile mockups for view payments though, so not sure if any of this is intentional. 

<img width="603" alt="Screen Shot 2020-06-29 at 12 10 40 PM" src="https://user-images.githubusercontent.com/15097156/86029884-c1f4c900-ba01-11ea-85a2-5aa4fa35ef49.png">

Reply:
Jim hasn't provided mobile mockups yet so this is as of yet largely unsettled. This will all get updated once the final mobile mockups are provided.